lib/NMISNG/Auth.pm

```diff
@@ -729,7 +702,7 @@ sub _ldap_verify
 	
 		$status = $ldap->bind("$dn",password=>"$p");
 		NMISNG::Util::logAuth("Auth::_ldap_verify, DEBUG: LDAP bind dn '$d' status ".$status->code()) if ($self->{debug});
-		if ($status->code eq 0) {
+		if (defined $status->code && $status->code == 0) {
```

Comment:
It's a bit nicer to call the function once when doing this ...

example code from net::ldap docs looks like this:
$mesg->code  and  die $mesg->error;          # check for errors